fastlane/actions/rocket.rb

```diff
@@ -0,0 +1,64 @@
+module Fastlane
+  module Actions
+    class RocketAction < Action
+      def self.run(params)
+        puts "
+        ____
+       /    \\
+      |      |
+      |      |
+      |      |
+       \\____/
+       |    |
+       |    |
+       |    |
+       |____|
+      {|    |}
+       |    |
+       |    |
+       | F  |
+       | A  |
+       | S  |
+       | T  |
+       | L  |
+       | A  |
+      /| N  |\\
```

Comment:
It's fairly standard for writing on a rocket to be bottom-aligned. See: Falcon 9's SpaceX sign, or "USA" on Saturn V. If you centered it across the whole rocket, it would be misaligned anyway after first stage separation.